contractcourt/channel_arbitrator.go

```diff
@@ -129,13 +129,31 @@ type ChannelArbitratorConfig struct {
 	ChainArbitratorConfig
 }
 
+// ReportOutputType describes the type of output that is being reported
+// on.
+type ReportOutputType uint8
+
+const (
+	// ReportOutputIncomingHtlc is an incoming hash time locked contract on
+	// the commitment tx.
+	ReportOutputIncomingHtlc ReportOutputType = iota
+
+	// ReportOutputOutgoingHtlc is an outgoing hash time locked contract on
+	// the commitment tx.
+	ReportOutputOutgoingHtlc
+
+	// ReportOutputDirect is an uncontested output on the commitment
+	// transaction paying to us directly.
+	ReportOutputDirect
```

Comment:
`ReportOutputToUs`?

Reply:
Everything that we report is (potentially) to us. But agreed that `Direct` is also pretty vague.

Reply:
Maybe `unencumbered` is what we use other places in the codebase?

Reply:
Renamed